src/screens/AirdropDashboard/components/AirdropFinished.tsx

```diff
@@ -5,39 +5,62 @@ import { Text, StyleSheet, View, TouchableOpacity, Linking } from 'react-native'
 
 import { images } from 'app/assets';
 import { AirdropStayTuned, AirdropWalletsList, Image } from 'app/components';
-import { RootStackParams, AirdropWalletDetails, Route } from 'app/consts';
+import { RootStackParams, Route, AirdropWalletCardData, Wallet } from 'app/consts';
 import { typography, palette } from 'app/styles';
 
+import { Error } from './Error';
+import { Loading } from './Loading';
+
 const i18n = require('../../../../loc');
 
 interface Props {
   navigation: CompositeNavigationProp<
     StackNavigationProp<RootStackParams, Route.MainTabStackNavigator>,
     StackNavigationProp<RootStackParams, Route.AirdropDashboard>
   >;
+  error: boolean;
+  loading: boolean;
+  wallets: Wallet[];
 }
 
 interface CallToActionProps {
-  wallet: AirdropWalletDetails;
+  data: AirdropWalletCardData;
   navigation: CompositeNavigationProp<
     StackNavigationProp<RootStackParams, Route.MainTabStackNavigator>,
     StackNavigationProp<RootStackParams, Route.AirdropDashboard>
   >;
 }
 
-const CallToAction: FC<CallToActionProps> = ({ wallet, navigation }) => {
-  const goToWalletDetails = (wallet: AirdropWalletDetails) => {
-    navigation.navigate(Route.AirdropFinishedWalletDetails, { balance: wallet.balance, name: wallet.name });
+const CallToAction: FC<CallToActionProps> = ({ data, navigation }) => {
+  const goToWalletDetails = (data: AirdropWalletCardData) => {
+    navigation.navigate(Route.AirdropFinishedWalletDetails, { balance: data.balance, label: data.label });
   };
 
   return (
-    <TouchableOpacity style={styles.arrowContainer} testID="forward-button" onPress={() => goToWalletDetails(wallet)}>
+    <TouchableOpacity style={styles.arrowContainer} testID="forward-button" onPress={() => goToWalletDetails(data)}>
       <Image style={styles.image} source={images.forwardArrow} />
     </TouchableOpacity>
   );
 };
 
-export const AirdropFinished: FC<Props> = ({ navigation }) => {
+export const AirdropFinished: FC<Props> = ({ navigation, error, loading, wallets }) => {
+  const renderContent = () => {
```

Comment:
should be a component, can be in this file

Reply:
done